client/src/app/core/core-services/operator.service.ts

```diff
@@ -26,12 +27,12 @@ function getUserCML(user: ViewUser): { [id: number]: string } | null {
         return null;
     }
 
-    const CML = {};
+    const permissionObject = {};
     const templateField = (user as any).committee_$_management_level;
     for (const replacement of templateField) {
-        CML[+replacement] = (user as any)[`committee_$${replacement}_management_level`];
+        permissionObject[+replacement] = (user as any)[`committee_$${replacement}_management_level`];
```

Comment:
You should be able to use `user.committee_manegement_level(id)`. This was a provisorium of mine due to the fact that the user didn't had a CML back then.